mason/breeders_toolbox/cross/crossingtrial_details.mas

```diff
@@ -13,6 +14,10 @@ $folder_id
 $identifier_prefix
 </%args>
 
+<& /breeders_toolbox/folder/folder_new.mas, project_type=>'crossing_experiment', breeding_program_name=>$breeding_program_name, breeding_program_id=>$breeding_program_id &>
```

Comment:
these two mason files are already loaded in crossing_trial.mas so you get DOM errors for duplicated IDs